ztp/source-cluster-crs/cluster-crs.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: siteconfig.Metadata.Name
+  labels:
+    name: siteconfig.Metadata.Name
+---
+apiVersion: extensions.hive.openshift.io/v1beta1
+kind: AgentClusterInstall
+metadata:
+  name: siteconfig.Spec.Clusters.ClusterName
+  namespace: siteconfig.Metadata.Name
+spec:
+  clusterDeploymentRef:
+    name: siteconfig.Spec.Clusters.ClusterName
+  imageSetRef:
+    name: siteconfig.Spec.ClusterImageSetNameRef
+  apiVIP: siteconfig.Spec.Clusters.ApiVIP
```

Comment:
For SNO api/ingress VIP are not required

Reply:
its part for the AgentClusterInstall CR, it is just will not be used with SNO